hardware_interface/include/hardware_interface/resource_manager.hpp

```diff
@@ -410,6 +424,12 @@ class HARDWARE_INTERFACE_PUBLIC ResourceManager
   mutable std::recursive_mutex claimed_command_interfaces_lock_;
   mutable std::recursive_mutex resources_lock_;
 
+  std::string interface_values_publisher_name_;
+  std::string interface_values_topic_name_;
+  rclcpp::Node::SharedPtr interface_value_publisher_node_;
+  rclcpp::Publisher<control_msgs::msg::DynamicInterfaceValues>::SharedPtr
```

Comment:
We have to use RealtimePublisher here, as we also use in every controller. Otherwise, will a break the real-time loop.